.github/workflows/github-actions.yml

```diff
@@ -405,6 +405,14 @@ jobs:
           flake8
           black --diff --check .
 
+      - name: Style check RST files
+        if: ${{ matrix.python-version == 3.7 }}
+        run: |
+          cd ${GITHUB_WORKSPACE}
+          ./scripts/rst-format.sh
```

Comment:
The rst-format.sh script is missing from this branch. Also, is the plan to format all rst files with this script before merging in this branch?

Reply:
Yes, I think we should format it. I didn't add the formatting script, I thought you added it -- how do I grab the `rst-format.sh`? It doesn't seem to be committed here.

Reply:
Oh did I push that commit? Let me check, I don't recall writing it, but I know i was looking into it on the drive up.